lib/dip/config.rb

```diff
@@ -3,6 +3,7 @@
 require "yaml"
 require "erb"
 require "pathname"
+require "json-schema"
```

Comment:
I am worried about this external dependency. Rather, the wrong version may load, since we cannot use the bundler here. This problem has not been solved for using Thor, and a new dependency can only make the situation worse.

Maybe something like this, but I'm not sure:
```ruby
gem "json-schema", ">= 5"
require "json-schema"
```

Reply:
Since the gem dependency is already specified in the gemspec, RubyGems ensures that the correct version of `json-schema` is loaded at runtime, even without the help of Bundler. The gemspec acts as the source of truth for dependencies, so the runtime will always use a compatible version of `json-schema` that satisfies the constraint (`~> 5`) defined there.  

If you'd like to verify, you can inspect the wrapper binary currently installed on your system by running:  

```sh  
cat $(which dip)  
```  

This will show how RubyGems handles dependency resolution for executables like this.